tests/Commands/ToggleDone.test.ts

```diff
@@ -78,7 +78,7 @@ function testToggleLineForOutOfRangeCursorPositions(
 
 describe('ToggleDone', () => {
     afterEach(() => {
-        resetSettings();
+        GlobalFilter.reset();
```

Comment:
Ah now, this is a change in behaviour. Previously it was resetting all settings, and now it's only resetting the GlobalFilter...

It may not matter, but it is maybe a pitfall...

Reply:
there are no calls here to `updateSettings()`, only `GlobalFilter.set()` hence the change.

Reply:
One argument in favour of your change to `GlobalFilter.reset();` is that it (rightly) hides the fact that the global filter is stored in settings.

I'll buy that!